src/ngraph_encapsulate_op.cc

```diff
@@ -191,6 +191,8 @@ class NGraphEncapsulateOp : public OpKernel {
         TensorDataToStream<uint8>(ostream, n_elements, data);
         break;
       case DT_UINT16:
+        TensorDataToStream<uint16>(ostream, n_elements, data);
+        break;
```

Comment:
since DT_UINT16 and DT_QUINT16 seem to map to uint16 only, we do not need these 2 lines. DT_UINT16's case should fall through to DT_QUINT16